indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -295,15 +295,20 @@ class BookingEditForm extends React.Component {
               initialVisibleMonth={() => moment(endDt)}
             />
           ) : (
-            <FinalDatePeriod
+            <FinalDateRangePicker
```

Comment:
That positioning of the dialog is... weird ;)

![image](https://github.com/user-attachments/assets/8c09bc38-9333-4d04-b7d3-96afbf093616)


To get the dialog, create a room booking and then edit it.

Reply:
Is that the whole viewport? Or is the dialog actually centered?

Reply:
It's indeed a centered dialog. 
![image](https://github.com/user-attachments/assets/35766c36-1d78-4a15-b425-02752388f55a)

Reply:
Eh. I bet that it's offset to the right and downward about as much as the dialog itself. 😁 I'll figure something out.

Reply:
![image](https://github.com/user-attachments/assets/5db614f9-b504-4d68-881e-e72c092a2907)

This fixes the issue, it appears. I'll add that to some page-specific stylesheet and we'll see if it works.